res/app/control-panes/info/info.pug

```diff
@@ -154,7 +154,7 @@
             div.infocard-table__item {{device.manufacturer}}
           div.infocard-table__row
             div.infocard-table__item(translate) Product
-            div.infocard-table__item {{device.name ? device.name : '-'}}
+            div.infocard-table__item {{device.model ? device.model : device.name}}
```

Comment:
I’d say if device.name exists and it must based on our infrastructure then it should be assigned into this column.
I’m ok to test simplified variant for product and model columns assigning only device.name…
There is no way to start service without this argument